includes/Settings/Settings_Page.php

```diff
@@ -125,6 +128,30 @@ public function render_page(): void {
 		<div class="wrap">
 			<h1><?php echo esc_html( get_admin_page_title() ); ?></h1>
 
+			<?php
+			// If we don't have proper credentials, show an error message and return early.
+			if ( ! has_ai_credentials() || ! has_valid_ai_credentials() ) {
```

Comment:
Can reduce this to just:
```suggestion
			if ( ! has_valid_ai_credentials() ) {
```

It implies the former.

Reply:
Ah, yeah, nice catch. Simplified that in https://github.com/WordPress/ai/pull/100/commits/a6e50c2b51e9aa7a53c7b9bd86f9130b3f494c42